addon/templates/components/uni-tooltip.hbs

```diff
@@ -0,0 +1,15 @@
+<span class="uni-tooltip__icon">
+    {{#if informationIcon}}
+        {{inline-svg "icons/information-icon"}}
+    {{else if questionIcon}}
+        {{inline-svg "icons/question-icon"}}
+    {{else}}
+        {{yield}}
```

Comment:
Hmm... I'm not very convinced but I accept it as I think it may be useful in some niche situations